website/content/docs/agent-and-proxy/agent/versions.mdx

```diff
@@ -2,21 +2,30 @@
 layout: docs
 page_title: Vault Agent Version Compatibility
 description: |-
-  Guidelines for running different versions of Agent and Server
+  Guidelines for running different versions of Agent and Vault server
 ---
 
-# Running different versions of agent and server
+# Running different versions of Agent and Vault server
 
 There is no requirement to run identical versions of Vault Agent and Vault Server.
-It is safe to run different versions, however you may not be able to take advantage of all the newest features in Vault if you do not upgrade to the most recent versions of Agent and Server. We recognize that this isn’t always possible, so we do support version mismatch as best as possible.
+It is safe to run different versions, however you may not be able to take
```

Comment:
"It is safe to run different versions. However, you may..."

Reply:
Good catch! Updating